apis/v1alpha1/httproute_types.go

```diff
@@ -203,6 +217,17 @@ type HTTPRouteRule struct {
 // * "RegularExpression"
 // * "ImplementationSpecific"
 //
+// For Prefix and Exact paths:
+// Every Path p_i must be syntactically valid:
+// 	- Must begin with the '/' character
+// 	- Must not contain consecutive '/' characters (e.g. /foo///, //).
+// 	- For prefix paths, a trailing '/' character in the Path is ignored,
+//      e.g. /abc and /abc/ specify the same match.
+//
+// If there is more than one potential match:
+// 	- Exact match is preferred to a Prefix match.
+//  - For multiple prefix matches, the longest Path p_i will be the matching path.
+//
```

Comment:
This can be made more specific.
See: https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/20190125-ingress-api-group.md#path-matching-semantics